docs/reverse_proxy.md

```diff
@@ -98,6 +98,37 @@ example.com:8448 {
   reverse_proxy http://localhost:8008
 }
 ```
+[Delegation](delegate.md) example:
+```
+(matrix-well-known-header) {
+    # Headers
+    header Access-Control-Allow-Origin "*"
+    header Access-Control-Allow-Methods "GET, POST, PUT, DELETE, OPTIONS"
+    header Access-Control-Allow-Headers "Origin, X-Requested-With, Content-Type, Accept, Authorization"
+    header Content-Type "application/json"
+}
+
+example.com {
+    handle /.well-known/matrix/server {
+        import matrix-well-known-header
+        respond `{"m.server":"matrix.example.com:8448"}`
```

Comment:
Why not just delegate to 443 and remove the `matrix.example.com:8448` handler entirely?

Reply:
Because I didn't want to deviate from the current docs as they were. Although I myself have this setup as you suggested myself, so I am definitely willing to update the docs on that too.
Not sure if I should add it in this PR too - if that's fine, then I'll gladly make those changes :)

Reply:
I've applied your suggestion just now.